### PR TITLE
Member & Plan DB improvements

### PR DESCRIPTION
**Pull request by chris18890**
This changes the member_plan_save & member_save mthods to use arrays when writing to the DB instead of declaring things explicitly - for member_plan_save this means the fields are hard-coded once instead of 6 times, which will make it easier in future if additional fields need to be added.

Also I've removed some old/commented out code & fixed the rendering of member names when deleting a membership

### CODE REVIEW

**Comment by chris18890** [comment]
Can I get a review @elplatt / @ramgarden?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Seltzer CRM 0.7.9 - An open source CRM for hackerspaces  
+Seltzer CRM 0.7.11 - An open source CRM for hackerspaces
```
Comment (ramgarden) [inline README.md:1]: Straight from 9 to 11?

File: crm/modules/member/data.inc.php [hunk 2]
```diff
@@ -289,17 +289,17 @@ function member_save ($member) {
...
-            SET `emergencyName`='$escaped[emergencyName]'
-                , `emergencyPhone`='$escaped[emergencyPhone]'
-                , `emergencyRelation`='$escaped[emergencyRelation]'
-                , `address1`='$escaped[address1]'
-                , `address2`='$escaped[address2]'
-                , `address3`='$escaped[address3]'
-                , `town_city`='$escaped[town_city]'
-                , `zipcode`='$escaped[zipcode]'
-            WHERE `cid`='$escaped[cid]'
+            SET " . implode(', ', $clauses) . "
```
Comment (ramgarden) [inline crm/modules/member/data.inc.php:301]: That is much better!

**Review by ramgarden — APPROVED** [review]
Good stuff for real.

File: README.md (hunk 1 above)
Comment (chris18890) [reply README.md:1]: the form update in 836650edd98e14fee06edf285d1060f9a88501d9 was .10 :)

**Pull request merged**